plugins/vcr.rst

```diff
@@ -0,0 +1,96 @@
+VCR Plugin
```

Comment:
i'd love the title to say in 2 words what it does, for those not making the association with video cassette recorders (feeling old saying this, but people starting to code might only have seen such a device in a museum :-D). that title will show up in the table of contents and should already indicate what the plugin does.

something like "VCR Plugin - Record and Replay Responses"